docs/data/data-grid/export/ExcelCustomExport.tsx

```diff
@@ -0,0 +1,326 @@
+import * as React from 'react';
+import 'exceljs';
```

Comment:
I would assume that fixing the code export dependencies would be enough to make this one work

```suggestion
```

Reply:
Not really, if there is no import of the library, `versions.exceljs` will be `undefined` and so it will not be added to libraries
The idea of this import is to add `exceljs` only in the demonstrations that require it

```js
newDeps.exceljs = versions.exceljs;
```

Reply:
I've noticed that `Edit in Codesandbox` throws an error:
> Uncaught TypeError: Cannot read properties of undefined (reading 'match')

Which refers to: https://github.com/mui/material-ui/blob/32192fbb6d511b79986731b3cf9dc09a27e0d3aa/docs/src/modules/utils/helpers.ts#L202